app/app/private-cloud/products/(product)/[licencePlate]/usage/page.tsx

```diff
@@ -0,0 +1,94 @@
+'use client';
+
+import { LoadingOverlay, Box } from '@mantine/core';
+import { useQuery } from '@tanstack/react-query';
+import { useState } from 'react';
+import { z } from 'zod';
+import FormSelect from '@/components/generic/select/FormSelect';
+import TableBodyMetrics from '@/components/table/TableBodyMetrics';
+import { IS_PROD } from '@/config';
+import createClientPage from '@/core/client-page';
+import { getPodUsageMetrics } from '@/services/backend/private-cloud/pod-usage-metrics';
+import { usePrivateProductState } from '@/states/global';
+
+const selectOptions = [
+  {
+    name: 'Production namespace',
+    value: 'prod',
+  },
+  {
+    name: 'Development namespace',
+    value: 'dev',
+  },
+  {
+    name: 'Tools namespace',
+    value: 'tools',
+  },
+  {
+    name: 'Test namespace',
+    value: 'test',
+  },
+];
+
+const pathParamSchema = z.object({
+  licencePlate: z.string(),
+});
+
+const privateCloudProductUsageMetrics = createClientPage({
+  roles: ['user'],
+  validations: { pathParams: pathParamSchema },
+});
+
+export default privateCloudProductUsageMetrics(({ pathParams, queryParams, session }) => {
+  const { licencePlate } = pathParams;
+  const [namespacePostfix, setNamespacePostfix] = useState('prod');
+  const [, privateSnap] = usePrivateProductState();
+
+  const { data = [], isLoading } = useQuery({
+    queryKey: [namespacePostfix, licencePlate],
+    queryFn: IS_PROD
+      ? () => getPodUsageMetrics(licencePlate, namespacePostfix, privateSnap.currentProduct?.cluster || 'silver')
+      : () => getPodUsageMetrics('f6ee34', namespacePostfix, 'klab'),
```

Comment:
let's move this logic to the backend with proper environment instead of hard code it.